features/shop/apply_correct_shipping_fee_with_taxes_on_order.feature

```diff
@@ -0,0 +1,41 @@
+@ui-cart
+Feature: Apply correct shipping fee with taxes on order
+    In order to pay proper amount for shipping
+    As a Customer
+    I want to have correct shipping fees and taxes applied to my order
+
+    Background:
+        Given the store is operating on a single channel
+        And there is "EU" zone containing all members of European Union
+        And there is rest of the world zone containing all other countries
+        And store ships to "France" and "Australia"
+        And default currency is "EUR"
+        And default tax zone is "EU"
+        And there is user "john@example.com" identified by "password123"
+        And store has "EU VAT" tax rate of 23% for "Clothes" within "EU" zone
+        And store has "Low tax" tax rate of 10% for "Clothes" for the rest of the world
+        And store has "Shipping EU VAT" tax rate of 23% for "Shipping Services" within "EU" zone
+        And store has "Shipping Low tax" tax rate of 10% for "Shipping Services World" for the rest of the world
```

Comment:
The tax category can also be "Shipping Services" here. Thanks to zone matching the proper tax will be applied for Australia anyway.